### ISSUE TITLE
Trying to launch a workspace with default namespace fails

### ISSUE THREAD

**Issue by Vafilor** [onepanelio/core#274]
Note: The namespace I use is `andrey`. By `default` I mean the namespace `default`, not `andrey` which is my.... uhh, default namespace.

Not sure if we need to address this issue. Does it make sense that the config for your default namespace would be in another namespace, namely `default`?

API error

```
ERRO[1376] getNamespaceConfig failed getting config map.  Error="configmaps \"onepanel\" not found" Namespace=default
ERRO[1376] Error parsing workflow.                       Error="configmaps \"onepanel\" not found" Namespace=default Workflow="&{0 0001-01-01 00:00:00 +0000 UTC    [{sys-workspace-action 0xc0029a60d0  <nil> <nil> [] false} {sys-resource-action 0xc0029a60e0  <nil> <nil> [] false} {sys-host 0xc0029a60f0  <nil> <nil> [] false} {sys-name 0xc000439db0  <nil> <nil> [] false} {sys-node-pool 0xc000439dc0  <nil> <nil> [] false} {sys-data-volume-size 0xc000439dd0  <nil> <nil> [] false} {sys-uid 0xc0029a6100  <nil> <nil> [] false}] []   <nil> <nil> 0xc0007500c0 map[]}"
ERRO[1376] finished unary call with code Unknown         error="configmaps \"onepanel\" not found" grpc.code=Unknown grpc.method=CreateWorkspace grpc.service=api.WorkspaceService grpc.start_time="2020-05-27T20:53:31-07:00" grpc.time_ms=725.592 span.kind=server system=grpc
```

**Comment by rushtehrani** [comment]
I have not been able to reproduce this in my EKS cluster so far.

**Comment by Vafilor** [comment]
@rushtehrani Gotcha, let me try again to see if I can do it again.

**Comment by rushtehrani** [comment]
@Vafilor moving this to v0.11.0 for verification on your end. @aleksandrmelnikov may also be able to help with this.

**Comment by aleksandrmelnikov** [comment]
Spun up a new cluster with v0.10.0
![image](https://user-images.githubusercontent.com/10407733/83927769-b84dad80-a742-11ea-879a-162725dfbaaf.png)

Of note, I used Azure cluter with 1.16.9 as the k8s version.

**Comment by rushtehrani** [comment]
Closing as this was confirmed to be working on both EKS and AKS
